.gitignore

```diff
@@ -1 +1,4 @@
 .venv
+
+# Default whiskers output directory
+themes
```

Comment:
I don't think we should ignore this and instead put the output of the templates inside of here, like so:

```
themes/
├─ cosmic-term/
├─ cosmic-settings/
```